canonical-web-and-design/hiring.canonical.com
# Jobs that do not have applications are removed from filter

## Done
Jobs that do not have applications are removed from filter
## QA
Setup:
- Add `DEBUG_USER_EMAIL=mark.shuttleworth@canonical.com` to `.env.local` file
- Run `dotrun` 

Validation:
- Verify job that does not have application is not displayed on the filter

## Issue
Fixes #122
